src/NUnitFramework/framework/Api/DefaultTestAssemblyBuilder.cs

```diff
@@ -79,7 +79,7 @@ public ITest Build(Assembly assembly, IDictionary<string, object> options)
 #else
             log.Debug("Loading {0} in AppDomain {1}", assembly.FullName, AppDomain.CurrentDomain.FriendlyName);
 #endif
-            
+
```

Comment:
No, it happens to everyone and if there are too many we just use look at the diff ignoring whitespace.